### PR TITLE
New referred user experience and altruistic giving language

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

File: src/components/ImpactDialog.js [hunk 1]
```diff
@@ -106,6 +106,30 @@ const ImpactDialog = ({
...
+        <>
+          <MuiDialogTitle disableTypography className={classes.root}>
+            <Typography variant="h6" className={classes.title}>
+              You've already made a difference!
+            </Typography>
+          </MuiDialogTitle>
+          <MuiDialogContent className={classes.walkMe}>
+            <Typography className={classes.justify}>
+              When your friend referred you to Tab For Cats, you each earned
+              five treats to give to shelter cats. Now, when you open a new tab
```
Comment (kmjennison) [inline src/components/ImpactDialog.js:120]: A few changes: "referred" is an internal term that's not very user-friendly; it's a bit long, and we already intro what treats do in the intro modal immediately previous to this; and I'd like to keep the "open a new tab" prompt to keep a clear next step. How about this:

### Open a new tab

Your friend started you off with 5 cat treats, which are crucial to getting shelter cats adopted.

Open a new tab now to earn your 6th treat! We'll track how many treats you've given on the top of the page:

**Review by kmjennison — CHANGES_REQUESTED** [review]
Copy changes. Ping me if you want to iterate/discuss.

File: src/components/ImpactDialog.js (hunk 1 above)
Comment (amaliwanag2) [reply src/components/ImpactDialog.js:120]: nice, yeah I like this better!

Comment (amaliwanag2) [reply src/components/ImpactDialog.js]: ok tried messing around with the text being two sections with spacing but it never looked as good as one code block IMO.  I think it's b/c the text in each section is never just 1 line and they're similar length.  I like the language a lot better

File: src/components/InviteFriendsIcon.js [hunk 2]
```diff
@@ -55,14 +54,11 @@ const InviteFriendsIcon = ({ user: { username, numUsersRecruited } }) => {
       >
         <div className={classes.popoverContent}>
           <Typography className={classes.titleSection}>
-            You've recruited{' '}
-            <span style={{ fontWeight: 'bold' }}>
-              {`${numUsersRecruited} friend${isPlural(numUsersRecruited)} `}
-            </span>{' '}
+            helping cats feels even better with friends
```
Comment (kmjennison) [inline src/components/InviteFriendsIcon.js:57]: This looks awkward to me. Capital "h", maybe bold title?

Screen Shot 2021-04-14 at 5 00 12 PM

**Review by kmjennison — APPROVED** [review]
LGTM 🍝

**Pull request merged**